cypress/integration/project-error-screen.spec.ts

```diff
@@ -0,0 +1,46 @@
+import capitalize from "lodash/capitalize";
+import mockProjects from "../fixtures/projects.json";
+
+describe("Project Error Screen", () => {
+  it("should checkout that error message exists when there is an error", () => {
+    const languageNames = ["React", "Node.js", "Python"];
+    const statusText = ["critical", "warning", "stable"];
+
+    cy.intercept("GET", "https://prolog-api.profy.dev/project", (req) =>
+      req.destroy()
+    ).as("getProjectsFailed");
+
+    // open projects page
+    cy.visit("http://localhost:3000/dashboard");
+
+    cy.wait(6000);
```

Comment:
Is this necessary since you wait for the request one line below anyways?

Reply:
I'm still new to Cypress. It shouldn't be necessary... but this one stumped me for a while; it kept skipping the `wait` on the line before. Cypress documentation indicates it should wait by all means, but I couldn't make the test pass unless I forced it to wait with the hardcoded number. Even 2-4 second waits were "flaky", so six it was 😅